tests/expand-manifest.jsonld

```diff
@@ -848,6 +852,22 @@
       "input": "expand/0114-in.jsonld",
       "expect": "expand/0114-out.jsonld",
       "option": {"specVersion": "json-ld-1.1"}
+    }, {
+      "@id": "#t0115",
+      "@type": ["jld:PositiveEvaluationTest", "jld:ExpandTest"],
+      "name": "Verifies that relative IRIs as properties with @vocab: '' in 1.0 are dropped",
+      "purpose": "Pathological relative property IRIs in 1.0",
```

Comment:
In 1.0 mode, this should be an error, not simply dropped/ignored, as the syntax says they MUST be absolute IRIs. It should be a NegativeEvaluationTest and generate an "invalid vocab mapping" error.